site/docs/capabilities/traffic/usage-based-ratelimiting.md

```diff
@@ -47,6 +47,12 @@ For model providers with OpenAI schema transformations (like AWS Bedrock), AI Ga
 
 ## Configuration
 
+:::tip
+
+- Envoy Gateway's rate limiting feature requires a redis deployment pointed by the Envoy Gateway installation time configuration.
+- See https://github.com/envoyproxy/ai-gateway/tree/main/examples/token_ratelimit for a complete example including redis deployment.
+  :::
```

Comment:
weird formatting by prettier cc @anuraaga

Reply:
Thanks for the note - interestingly, docusaurus even mentions this issue, so it's a well supported bug 😂 

https://docusaurus.io/docs/next/markdown-features/admonitions#usage-with-prettier

I guess the real fix would be when they support blockquote style admonitions so then it can render well both in docusaurus and on GitHub, but not yet I guess :(

https://docusaurus.io/feature-requests/p/support-blockquote-style-admonitions